### PR TITLE
Enforce linking to spec for method implementations via macros

### PR DESCRIPTION
**Pull request by frewsxcv**
[Review on Reviewable](https://reviewable.io/reviews/servo/servo/7483)

### CODE REVIEW

File: components/script/dom/worker.rs [hunk 1]
```diff
@@ -162,6 +162,7 @@ impl WorkerMethods for Worker {
         Ok(())
     }
 
+    // https://html.spec.whatwg.org/multipage/#handler-dedicatedworkerglobalscope-onmessage
     event_handler!(message, GetOnmessage, SetOnmessage);
     event_handler!(error, GetOnerror, SetOnerror);
```
Comment (jdm) [inline components/script/dom/worker.rs]: Curious that this doesn't get flagged.

Comment (frewsxcv) [inline components/script/dom/worker.rs]: Hm. I'll look into it

Comment (frewsxcv) [inline components/script/dom/worker.rs]: https://github.com/frewsxcv/servo/blob/macro-spec-links/python/tidy.py#L338

this checks two lines up for some reason...

Comment (frewsxcv) [inline components/script/dom/worker.rs]: for cases like this i think. i'll work around it

https://github.com/servo/servo/pull/6699/files#diff-ffcaa7c334fe34498bc7362e376be445R466

Comment (jdm) [inline components/script/dom/worker.rs]: Heh, you're right. I forgot about that.

Comment (frewsxcv) [inline components/script/dom/worker.rs]: Fixed in the latest force push

**Comment by jdm** [comment]
@bors-servo: r+
Excellent!

**Pull request merged**